src/main/java/com/aws/greengrass/deployment/DeploymentService.java

```diff
@@ -745,7 +756,8 @@ private DefaultDeploymentTask createDefaultNewDeployment(Deployment deployment)
     private DeploymentDocument parseAndValidateJobDocument(Deployment deployment) throws InvalidRequestException {
         String jobDocumentString = deployment.getDeploymentDocument();
         if (Utils.isEmpty(jobDocumentString)) {
-            throw new InvalidRequestException("Job document cannot be empty");
+            throw new InvalidRequestException("deployment document cannot be empty",
```

Comment:
nit: Deployment document - although some error messages say "job document" - when do you use deployment vs job?

Reply:
changed to deployment document